src/module_2/module_2.md

```diff
@@ -0,0 +1,3512 @@
+      <th>id</th>
+      <th>user_id</th>
+      <th>created_at</th>
+      <th>order_date</th>
+      <th>user_order_seq</th>
+      <th>ordered_items</th>
+      <th>variant_id</th>
+      <th>price</th>
+      <th>compare_at_price</th>
+      <th>vendor</th>
+      <th>product_type</th>
+      <th>tags</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>107953</th>
+      <td>4026732413060</td>
+      <td>faaad4bc113fe7e0a888a0d3535877d3c692f591882c25...</td>
+      <td>2022-03-14 00:24:59</td>
+      <td>2022-03-14</td>
+      <td>3</td>
+      <td>34537169453188</td>
+      <td>34537169453188</td>
+      <td>1</td>
+      <td>1</td>
+      <td>whiskas</td>
+      <td>cat-food</td>
+      <td>[cat-treats]</td>
+    </tr>
+    <tr>
+      <th>107954</th>
+      <td>4026732413060</td>
+      <td>faaad4bc113fe7e0a888a0d3535877d3c692f591882c25...</td>
+      <td>2022-03-14 00:24:59</td>
+      <td>2022-03-14</td>
+      <td>3</td>
+      <td>39368665432196</td>
+      <td>39368665432196</td>
+      <td>5</td>
+      <td>5</td>
+      <td>whiskas</td>
+      <td>cat-food</td>
+      <td>[wet-cat-food]</td>
+    </tr>
+    <tr>
+      <th>107955</th>
+      <td>4026732413060</td>
+      <td>faaad4bc113fe7e0a888a0d3535877d3c692f591882c25...</td>
+      <td>2022-03-14 00:24:59</td>
+      <td>2022-03-14</td>
+      <td>3</td>
+      <td>39403033428100</td>
+      <td>39403033428100</td>
+      <td>3</td>
+      <td>3</td>
+      <td>napolina</td>
+      <td>tins-packaged-foods</td>
+      <td>[beans-pulses-lentils, discontinue, gluten-fre...</td>
+    </tr>
+    <tr>
+      <th>107956</th>
+      <td>4026732413060</td>
+      <td>faaad4bc113fe7e0a888a0d3535877d3c692f591882c25...</td>
+      <td>2022-03-14 00:24:59</td>
+      <td>2022-03-14</td>
+      <td>3</td>
+      <td>39459281272964</td>
+      <td>39459281272964</td>
+      <td>4</td>
+      <td>9</td>
+      <td>cif</td>
+      <td>cleaning-products</td>
+      <td>[bathroom-limescale-cleaner, refills]</td>
+    </tr>
+    <tr>
+      <th>107957</th>
+      <td>4026732413060</td>
+      <td>faaad4bc113fe7e0a888a0d3535877d3c692f591882c25...</td>
+      <td>2022-03-14 00:24:59</td>
+      <td>2022-03-14</td>
+      <td>3</td>
+      <td>39830155034756</td>
+      <td>39830155034756</td>
+      <td>4</td>
+      <td>9</td>
+      <td>cif</td>
+      <td>cleaning-products</td>
+      <td>[kitchen-oven-cleaner, refills]</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+Now that we have the orders with products info, we can add additional info for each user (contained in df_users), about its regular wish list product (df_regulars), and about other items they dropped from their basket (df_abandoned)
+
+What we would do for continuing inspecting orders is plot the data to know how many times a user ordered something (only with orders)
+
+
+```python
+# df_orders['user_order_seq'].value_counts().plot(kind='bar')
+sns.countplot(data=df_orders_unrolled, x='user_order_seq', color='red')
```

Comment:
This discussion probably doesn't belong here, but any reason to use seaborn? I don't normally like it. Check out `altair` or `plotnine` (the adaptation of `ggplot` for Python) which are declarative viz libraries.